index.html

```diff
@@ -12,39 +12,175 @@
     <!--This is the header section-->
     <header>
         <nav>
-            <a href="#"><img src="mylogo.png" alt="Logo" id="myLogo"></a>
+            <a href="#"><img src="req/mylogo.png" alt="Logo" id="myLogo"></a>
             <!--Nav bar button-->
-            <img src="Union.png" alt="Logo" id="logo">
+            <img src="req/Union.png" alt="Logo" id="logo">
         </nav>
     </header>
     <!--This is Headline-->
     <section class="headline">
-        <div id="bigText"><h1>I’m Adil
-            Glad to see you!</h1></div>
-        <div id="smallText">
-            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div><p id="bigText">I’m Adil<br>
+            Glad to <br>see you!</p></div>
+        <div>
+            <p class="smallText">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
         </div>
         <div class="socialmedia">
-            <div id="supportingText">LET'S CONNECT</div>
+            <div class="supportingText">LET'S CONNECT</div>
```

Comment:
- [ ] [OPTIONAL] For semantic reasons, please consider using any of `h2-h6` tag instead of a `div`. Remember that `div`s have no special meaning other than for grouping of elements. As such, it best to always use alternatively correct tags where and when possible 🔖